docs/assets/scss/_local-overrides.scss

```
@@ -0,0 +1,233 @@
+// Project-local overrides layered on top of gooddata-docs-theme.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)